.chloggen/metrics_builder_constructor_ms_to_mbc.yaml

```
@@ -0,0 +1,20 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement # breaking?
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: mdatagen
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Changes MetricsBuilder constructor from MetricsSettings to MetricsBuilderConfig
+
+# One or more tracking issues related to the change
+issues: []
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  Currently, the only way to add configurative functionality to mdatagen is by using a bunch of WithBlah options to the constructor.
```

```suggestion
  Currently, the only way to add configurative functionality to mdatagen is by using a bunch of WithResourceAttributesSettings options to the constructor.
```

Please remove `WithResourceAttributesSettings`, it shouldn't be needed anymore